.eslintrc.js

```diff
@@ -33,5 +33,6 @@ module.exports = {
     "curly": "error",
     "no-console": "warn",
     "semi": "error",
+    "space-before-function-paren": [2, "never"],
```

Comment:
Nit: Me thinks this should be `["error", "never"]` for consistency.

I think this is coming from code w/ anonymous functions like `async()`.
Curiously, this seems to be coming from some change in **eslint-plugin-mozilla**, if I'm looking at this correctly.

Reply:
If it's all async arrow functions that it's grumpy about, we should fix the code. I only avoided putting a space there to make the linter happy in the first place.

Reply:
In that case, running `npm run lint -- --fix` should auto fix all these to these:

```diff
-openDataStore().then(async(ds) => {
+openDataStore().then(async (ds) => {
```